message/pool/message.go

```diff
@@ -99,7 +99,7 @@ func (r *Message) SetType(typ message.Type) {
 
 // UpsertType set value only when origin value is invalid. Only 0 to 2^8-1 values are valid.
 func (r *Message) UpsertType(typ message.Type) {
-	if r.msg.Type >= 0 && r.msg.Type < math.MaxUint8 {
+	if r.msg.Type >= 0 && r.msg.Type <= math.MaxUint8 {
```

Comment:
Please, could you create `message.ValidateType(mid int32) bool`, it will be used at both places, as asked for by @mniestroj?